arrow/parser.py

```diff
@@ -337,7 +504,8 @@ def _parse_token(self, token, value, parts):
             parts["year"] = 1900 + value if value > 68 else 2000 + value
 
         elif token in ["MMMM", "MMM"]:
-            parts["month"] = self.locale.month_number(value.lower())
+            # FIXME: month_number() is nullable
+            parts["month"] = self.locale.month_number(value.lower())  # type: ignore
```

Comment:
@isac322 Is this FIXME good to be removed?

Reply:
@jadchaar  Hmmmm. I'm not sure `Locale.month_number()` always return `str`.